src/core/Akka/Actor/IActorContext.cs

```diff
@@ -135,6 +135,11 @@ public interface IActorContext : IActorRefFactory, ICanWatch
         /// <param name="timeout">The timeout. Pass in <c>null</c> to switch off this feature.</param>
         void SetReceiveTimeout(TimeSpan? timeout);
 
+        /// <summary>
+        /// Gets the inactivity deadline timeout set using <see cref="SetReceiveTimeout"/>.
+        /// </summary>
+        TimeSpan? ReceiveTimeout { get; }
```

Comment:
This does not version well, Adding members to interfaces.
But..
It is a member in Scala and thus belongs here. and there is currently no way of checking if receive timeout has been set to a value.
It is also very unlikely that there are custom implementations of the interface out there that does not inherit `ActorCell`.

Thoughts on how to handle this? there are probably more places where we haven't yet fully ported the Scala code and need to add more members to support new features.